db/migrate/20221004130844_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.integer :preparationTime
+      t.integer :cookingTime
```

Comment:
It is better to use under scorer (_) separated column names, like preparation_time and cooking_time.
It is better to add other migration to rename this column.
You can follow this example
rails g migration rename_preparationTime_to_preparation_time
class RenamePreparationTimeToPreparationTime < ActiveRecord::Migration
  def change
  end
end

Modify it as follows:

class RenamePreparationToPreparationTime < ActiveRecord::Migration
  def change
    rename_column :recipes, :preparationTime, :preparation_time
  end
end